test_driver/app_test.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:MobileOne/pages/authentication-page.dart';
+/*import 'package:MobileOne/pages/authentication-page.dart';
```

Comment:
no no no, it's too easy to comment a test like this. You have to make a choice : 
1- Delete the test because it is outdated
2- Fix the test